config/flags.go

```diff
@@ -0,0 +1,220 @@
+package config
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
+)
+
+type buxFlags struct {
```

Comment:
Why we need this struct? Couldn't we use `pflag.FlagSet` directly?

Reply:
No, we can't for some reason.
`type buxFlags pflag.FlagSet` doesn't work. No idea why. 

And we need this custom type to generate methods against it.